cli/inference_cli/lib/container_adapter.py

```diff
@@ -0,0 +1,112 @@
+import subprocess
+
+import docker
+import typer
+
+docker_client = docker.from_env()
+
+
+def ask_user_to_kill_container(c):
+    name = c.attrs.get("Name", "")
+    env_vars = c.attrs.get("Config", {}).get("Env", {})
+    port = 9001
+    for var in env_vars:
+        if var.startswith("PORT="):
+            port = var.split("=")[1]
+    should_delete = typer.confirm(
+        f" An inference server is already running in container {name} on port {port}. Are you sure you want to delete it?"
+    )
+    return should_delete
+
+
+def is_inference_server_container(container):
+    image_tags = container.image.tags
+    for t in image_tags:
+        if t.startswith("roboflow/roboflow-inference-server"):
```

Comment:
Sidenote I think we should change these dockers to live at roboflow/inference.

Reply:
Added a ticket for this.